src/test/resources/setup.sh

```diff
@@ -36,11 +36,16 @@ if [ -z "${OPENSHIFT_MASTER}" ]; then
 fi
 
 #Configure the SYNDESIS_E2E_SECRET
-if [ -n ${SYNDESIS_E2E_SECRET} ]; then
-    GITHUB_OAUTH_CLIENT_ID=${GITHUB_E2E_OAUTH_CLIENT_ID}
-    GITHUB_OAUTH_CLIENT_SECRET=${GITHUB_E2E_OAUTH_CLIENT_SECRET}
+if [ ! -z ${SYNDESIS_E2E_SECRET} ]; then
```

Comment:
Variables should always be quoted in conditionals and prefer simple `-n` to `! -z`